core/chains/evm/txmgr/eth_confirmer_test.go

```diff
@@ -974,22 +974,28 @@ func TestEthConfirmer_CheckConfirmedMissingReceipt(t *testing.T) {
 	etx2 := cltest.MustInsertConfirmedMissingReceiptEthTxWithLegacyAttempt(
 		t, borm, 2, 1, originalBroadcastAt, fromAddress)
 	attempt2_1 := etx2.EthTxAttempts[0]
+	etx3 := cltest.MustInsertConfirmedMissingReceiptEthTxWithLegacyAttempt(
+		t, borm, 3, 1, originalBroadcastAt, fromAddress)
+	attempt3_1 := etx3.EthTxAttempts[0]
 
 	ethClient.On("BatchCallContextAll", mock.Anything, mock.MatchedBy(func(b []rpc.BatchElem) bool {
-		return len(b) == 3 &&
+		return len(b) == 4 &&
 			cltest.BatchElemMatchesParams(b[0], hexutil.Encode(attempt0_2.SignedRawTx), "eth_sendRawTransaction") &&
 			cltest.BatchElemMatchesParams(b[1], hexutil.Encode(attempt1_2.SignedRawTx), "eth_sendRawTransaction") &&
-			cltest.BatchElemMatchesParams(b[2], hexutil.Encode(attempt2_1.SignedRawTx), "eth_sendRawTransaction")
+			cltest.BatchElemMatchesParams(b[2], hexutil.Encode(attempt2_1.SignedRawTx), "eth_sendRawTransaction") &&
+			cltest.BatchElemMatchesParams(b[3], hexutil.Encode(attempt3_1.SignedRawTx), "eth_sendRawTransaction")
 	})).Return(nil).Run(func(args mock.Arguments) {
 		elems := args.Get(1).([]rpc.BatchElem)
 		// First transaction confirmed
 		elems[0].Error = errors.New("nonce too low")
 		elems[1].Error = errors.New("transaction underpriced")
 		elems[2].Error = nil
+		elems[3].Error = errors.New("transaction already finalized")
 	}).Once()
 	println("Test idx: ", etx0.ID)
```

Comment:
Oh, lines 995-998 are actually debug print statements left there by mistake. 
Can you please remove these?

Reply:
yes, I will